Make error

Ubuntu 18.04.1 LTS x86_64

git clone https://github.com/hashcat/hashcat.git
make

And I am getting:

src/modules/module_33501.c:62:36: error: initializer element is not constant
 static const u32   PW_LEN        = RC4_KEY_BITS / 8; // 9
                                    ^~~~~~~~~~~~
src/Makefile:709: recipe for target 'modules/module_33501.so' failed
make: *** [modules/module_33501.so] Error 1
make: *** Waiting for unfinished jobs....

--- a/src/modules/module_33500.c
+++ b/src/modules/module_33500.c
@@ -58,8 +58,8 @@ typedef struct rc4
 } rc4_t;
 
 static const char *SIGNATURE_RC4 = "$rc4$";
-static const u32   RC4_KEY_BITS  = 40;
-static const u32   PW_LEN        = RC4_KEY_BITS / 8; // 5
+#define RC4_KEY_BITS 40
+#define PW_LEN (RC4_KEY_BITS / 8) // 5
 
 bool module_unstable_warning (MAYBE_UNUSED const hashconfig_t *hashconfig, MAYBE_UNUSED const user_options_t *user_options, MAYBE_UNUSED const user_options_extra_t *user_options_extra, MAYBE_UNUSED const hc_device_param_t *device_param)
 {

--- a/src/modules/module_33501.c
+++ b/src/modules/module_33501.c
@@ -58,8 +58,8 @@ typedef struct rc4
 } rc4_t;
 
 static const char *SIGNATURE_RC4 = "$rc4$";
-static const u32   RC4_KEY_BITS  = 72;
-static const u32   PW_LEN        = RC4_KEY_BITS / 8; // 9
+#define RC4_KEY_BITS 72
+#define PW_LEN (RC4_KEY_BITS / 8) // 9
 
 bool module_unstable_warning (MAYBE_UNUSED const hashconfig_t *hashconfig, MAYBE_UNUSED const user_options_t *user_options, MAYBE_UNUSED const user_options_extra_t *user_options_extra, MAYBE_UNUSED const hc_device_param_t *device_param)
 {

--- a/src/modules/module_33502.c
+++ b/src/modules/module_33502.c
@@ -58,8 +58,8 @@ typedef struct rc4
 } rc4_t;
 
 static const char *SIGNATURE_RC4 = "$rc4$";
-static const u32   RC4_KEY_BITS  = 104;
-static const u32   PW_LEN        = RC4_KEY_BITS / 8; // 13
+#define RC4_KEY_BITS 104
+#define PW_LEN (RC4_KEY_BITS / 8) // 13
 
 bool module_unstable_warning (MAYBE_UNUSED const hashconfig_t *hashconfig, MAYBE_UNUSED const user_options_t *user_options, MAYBE_UNUSED const user_options_extra_t *user_options_extra, MAYBE_UNUSED const hc_device_param_t *device_param)
 {

